src/services/database.ts

```diff
@@ -53,40 +55,46 @@ const getPageIdQuery = (projectName: string, branch: string) => {
 export const initDb = (client: MongoClient) => {
   const dbName = process.env.SNOOTY_DB_NAME ?? 'snooty_dev';
   db = client.db(dbName);
+  const prodDbName = process.env.SNOOTY_PROD_DB_NAME ?? 'snooty_dev';
```

Comment:
(Non-blocking suggestion) It might be beneficial to include a comment that `prodDbName` is representative of production deployments. I can imagine it being confusing when lacking full context on terminology here (prod vs. prd vs. dotcomprd)